### ISSUE TITLE
Elevated circulating S-adenosyl-L-methionine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8154]
**Preferred term label:**
Elevated circulating S-adenosyl-L-methionine concentration

**Synonyms**
AdoMet

**Definition (free text, please give PubMed ID)**
Elevated concentration S-adenosyl-L-methionine oin the blood circulation

**Parent term (use hpo.jax.org/app)**

Abnormal circulating non-proteinogenic amino acid concentration HP:0033109
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614300 
PMID: 21963049  6/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
